pom.xml

```diff
@@ -263,4 +263,26 @@ limitations under the License.
       </plugin>
     </plugins>
   </build>
+  <profiles>
+    <profile>
+      <id>useBigtableSnapshots</id>
+      <!-- use the snapshot veneer artifact to run the tests-->
+      <properties>
+        <bigtable.version>LATEST</bigtable.version>
+      </properties>
+
+      <repositories>
+        <repository>
+          <id>sonatype-snapshots</id>
+          <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+          <releases>
+            <enabled>false</enabled>
+          </releases>
+          <snapshots>
+            <enabled>true</enabled>
+          </snapshots>
+        </repository>
+      </repositories>
+    </profile>
+  </profiles>
```

Comment:
why is this necessary?

Reply:
I am not sure. I think Cheng added it for enabling integ tests. @kolea2 do you know?

Reply:
You can revert this. This is getting pulled in from https://github.com/googleapis/java-bigtable-hbase/pull/2535, which isn't configured to run on this branch.

Reply:
@vermas2012 friendly ping :)

Reply:
removed.